plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -71,7 +71,7 @@ func init() {
 	addControllerRole(rbac.ClusterRole{
 		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "cronjob-controller"},
 		Rules: []rbac.PolicyRule{
-			rbac.NewRule("get", "list", "watch", "update").Groups(batchGroup).Resources("cronjobs").RuleOrDie(),
+			rbac.NewRule("get", "list", "watch", "update", "delete").Groups(batchGroup).Resources("cronjobs").RuleOrDie(),
```

Comment:
this commit needs repicking... this isn't the permission we ended up granting upstream

Reply:
There are 2 modifying this:
* b38481343c - UPSTREAM: 49133: add controller permissions to set blockOwnerDeletion (6 days ago) <David Eads>
* 6b69907f2e - UPSTREAM: 49133: update permissions to allow block owner deletion (7 days ago) <deads2k>

We could squash them into one.

Reply:
yeah

Reply:
squashed